acq4/devices/PatchPipette/states.py

```diff
@@ -1065,9 +1065,10 @@ class PatchPipetteResealState(PatchPipetteState):
         'retractionSpeed': 0.3e-6,
         'resealTimeout': 10 * 60,
         'numTestPulseAverage': 3,
-        'fallbackState': 'whole cell',
+        'fallbackState': 'bath',
```

Comment:
'whole cell' is the correct fallback state -- we are starting from whole cell; if anything unexpected happens while resealing, then we should return to whole cell. Falling back to bath means that a bug in the reseal procedure causes the cell to be instantly killed.

Reply:
I was told that when reseal fails, the pipette is in the bath, no longer attached to the cell.